core/io/file_access_pack.cpp

```diff
@@ -241,6 +241,7 @@ bool PackedSourcePCK::try_open_pack(const String &p_path, bool p_replace_files,
 
 		String path;
 		path.parse_utf8(cs.ptr());
+		path = String("res://") + p_prefix + path.replace_first("res://", "");
```

Comment:
Could this maybe use `String.path_join()` instead of `+` to append the prefix? That way, there's no need to remember the trailing slash when specifying the prefix.

Reply:
i was wondering the same, but decided against it for the "first draft" because one might simply want to prefix the contained file*names* instead of specifying a whole folder. no idea which way would be better honestly, one is "more noob friendly" (not sure how important for such a "lowlevel feature"), the other is "more flexible" so i arbitrarily chose it.

Reply:
yeah i tested it a bit more, and the side effects of prefixing with just a filename part are just too weird for pretty much every situation... pushing the change now :P